core/src/utils/keyboard/keyboard-controller.ts

```diff
@@ -1,33 +1,98 @@
-import { win } from '../window';
+import { raf } from '@utils/helpers';
+import { win } from '@utils/window';
+
+import { KeyboardResize, Keyboard } from '../native/keyboard';
```

Comment:
This gave me a strange build error when I tried to do `@utils/native/keyboard`:

```
[ ERROR ]  Rollup: Unresolved Import
           Could not resolve '../native/keyboa' from ./src/utils/keyboard/keyboard-controller.ts
```

Reply:
Was this resolved? I wasn't able to reproduce any build errors.

Reply:
I'm still getting the issue locally when doing `npm run start`:

```
[ ERROR ]  Rollup: Unresolved Import
           Could not resolve '../native/keyboa' from
           ./src/utils/keyboard/keyboard-controller.ts
```

Reply:
I'm also not able to reproduce any issues when running `npm run start`. Does it happen as soon as the command is entered or does it show when on a specific page?

Reply:
I'm getting the error as soon as I run `npm start`, or at least a bit later once the build has finished. I did also try `rm ./dist` and `npm install` in case I had some build cache or oudated Stencil issues, but the error's still occuring. Odd 🤔

Reply:
I can ask the Stencil team about this during our sync today, but I don't think it's worth holding up this PR over.

Reply:
Super weird, I'll be curious to know the outcome.

Reply:
Spoke with Ryan, and he thinks the compiler may be expecting to see a `ts` extension since only 2 characters are being cut off (i.e. the length of the file extension). Tanner is working on a similar bug, and Ryan had me try dev build with Tanner's proposed fix. However, the proposed fix did not change the behavior here.

Reply:
For posterity, the fix for this is tied to https://github.com/ionic-team/stencil/pull/4481

Reply:
Fix landed/released in Stencil v3.4.1 - https://github.com/ionic-team/stencil/releases/tag/v3.4.1